core/block/import/pb/gallery.go

```diff
@@ -0,0 +1,92 @@
+package pb
+
+import (
+	"github.com/globalsign/mgo/bson"
+	"github.com/samber/lo"
+
+	"github.com/anyproto/anytype-heart/core/block/collection"
+	"github.com/anyproto/anytype-heart/core/block/editor/state"
+	widgets "github.com/anyproto/anytype-heart/core/block/editor/widget"
+	"github.com/anyproto/anytype-heart/core/block/import/converter"
+	"github.com/anyproto/anytype-heart/core/block/simple"
+	"github.com/anyproto/anytype-heart/pb"
+	"github.com/anyproto/anytype-heart/pkg/lib/bundle"
+	"github.com/anyproto/anytype-heart/pkg/lib/pb/model"
+	"github.com/anyproto/anytype-heart/util/pbtypes"
+)
+
+type GalleryImport struct {
+	service *collection.Service
+}
+
+func NewGalleryImport(service *collection.Service) *GalleryImport {
+	return &GalleryImport{service: service}
+}
+
+func (g *GalleryImport) ProvideCollection(_ []*converter.Snapshot, widget *converter.Snapshot, _ map[string]string, params *pb.RpcObjectImportRequestPbParams, workspaceSnapshot *converter.Snapshot) (*converter.Snapshot, error) {
```

Comment:
Again I suggest here very-short-line-but-long-to-read-advice function length :upside_down_face: 
```
func (g *GalleryImport) ProvideCollection(
    _ []*converter.Snapshot, 
    widget *converter.Snapshot,
     _ map[string]string, 
    params *pb.RpcObjectImportRequestPbParams, 
    workspaceSnapshot *converter.Snapshot,
) (*converter.Snapshot, error) {
```